policyengine_us/parameters/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/amount/not_separate.yaml

```diff
@@ -0,0 +1,19 @@
+description: Hawaii provides this amount under dependent care benefit for non-separate filler in line 11.
+values:
+  2018-01-01: 5_000
+metadata:
+  unit: currency-USD
+  label: Hawaii dependent care benefit amount for non-separate
+  reference:
+    - title: Hawaii Income Tax Law, Chapter 235, Section 235-55.6, (a)(1)
```

Comment:
not sure if I can see the parameter in the source

Reply:
It is in the line 11 of https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2